rest/taskana-rest-spring/src/main/java/pro/taskana/workbasket/rest/WorkbasketQueryFilterParameter.java

```diff
@@ -0,0 +1,119 @@
+package pro.taskana.workbasket.rest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.beans.ConstructorProperties;
+import java.util.Optional;
+
+import pro.taskana.common.rest.QueryParameter;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketQuery;
+import pro.taskana.workbasket.api.WorkbasketType;
+
+public class WorkbasketQueryFilterParameter implements QueryParameter<WorkbasketQuery, Void> {
+
+  /** Filter by the name of the workbasket. This is an exact match. */
+  private final String[] name;
+
+  /**
+   * Filter by the name of the workbasket. This results into a substring search. (% is appended to
+   * the front and end of the requested value). Further SQL "Like" wildcard characters will be
+   * resolved correctly.
+   */
+  @JsonProperty("name-like")
+  private final String[] nameLike;
+
+  /** Filter by the key of the workbasket. This is an exact match. */
+  private final String[] key;
+
+  /**
+   * Filter by the key of the workbasket. This results into a substring search. (% is appended to
+   * the front and end of the requested value). Further SQL "Like" wildcard characters will be
+   * resolved correctly.
+   */
+  @JsonProperty("key-like")
+  private final String[] keyLike;
+
+  /** Filter by the owner of the workbasket. This is an exact match. */
+  private final String[] owner;
+
+  /**
+   * Filter by the owner of the workbasket. This results into a substring search. (% is appended to
```

Comment:
into -> in, appended to the front and end ->appended at the beginning and end? Also for the following below